apps/src/code-studio/components/progress/teacherPanel/StudentTable.jsx

```diff
@@ -96,8 +95,7 @@ class StudentTable extends React.Component {
                   )}
                   <div style={styles.name}>
                     {student.name}
-                    {!!DCDO.get('family-name-features', false) &&
-                      ` ${student.familyName || ''}`}
+                    {` ${student.familyName || ''}`}
```

Comment:
nit: i think this could be cut down even further now to 
```js
{`${student.name} ${student.familyName || ''}`}
```

Reply:
Ahh good catch. Thanks!